packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -1,33 +1,249 @@
 /*
- * Copyright (c) 2017, salesforce.com, inc.
- * All rights reserved.
- * Licensed under the BSD 3-Clause license.
- * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
- */
+* Copyright (c) 2017, salesforce.com, inc.
+* All rights reserved.
+* Licensed under the BSD 3-Clause license.
+* For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+*/
 import * as fs from 'fs';
+import * as mkdirp from 'mkdirp';
 import * as path from 'path';
-import { SObjectCategory, SObjectDescribe } from '../describe';
+import {
+  ChildRelationship,
+  Field,
+  SObject,
+  SObjectCategory,
+  SObjectDescribe
+} from '../describe';
 
 export class FauxClassGenerator {
```

Comment:
How do you feel about creating mini classes that generate different parts of the class stub? 
One part for the class itself.
One part for the fields. 
All the classes can be contained inside this single file.
But it might make it more clear and easier to test instead of just doing everything in one class.

Reply:
IMHO it feels odd,  there really isn't anything in those classes except for a function, not what I think of when I think of a class...    I agree that methods/functions shouldn't be long at all - but I"m not so certain that the class size is fundamental.    Seems more important to have class coherence...      And yes, there are some tests still missing

Reply:
Right now your tests verify both the generated text and also the creation of the folders.
Might be better to be able to verify them independently.

Reply:
OK. Just checking with you. I don't have a strong opinion of it now. We can revisit if the class's responsibilities get bigger.